client/src/pages/SignIn.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { User } from '../../../types/User';
+
+export default function SignIn(): JSX.Element {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  async function loginUser(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    const user = {
```

Comment:
This request user should be typed, specially because it is being sent to the server. Take as example the user page I did

Reply:
yes it is not complete yet, I am working on it, thank you